branding/ovirt-common.css

```diff
@@ -0,0 +1,12 @@
+.login-pf body{background-color:#131822;background-image:none}.login-pf .ovirt-container{font-size:13px}.login-pf .container{width:100%}.navbar-pf-vertical{border-top:4px solid #00659c !important}.nav-pf-vertical,.nav-pf-secondary-nav,.nav-pf-tertiary-nav{top:62px}.container-pad{padding-left:100px !important;padding-top:50px !important;padding-bottom:60px !important}#welcome-section{position:absolute;top:-76px;left:100px}#welcome-section-login{position:absolute;top:-50px;left:100px}#welcome-band-top{margin-top:15px}#welcome-title{position:relative;top:-4px;font-size:14px;font-weight:400;float:left}#welcome-version-text{position:relative;top:8px;margin-left:30px;color:#bbb;float:left}#welcome-app-title{position:relative;top:-3px;left:10px;float:left;font-size:18px;font-weight:600}@media (max-width: 768px){#welcome-app-title{display:none}}#welcome-app-title .pipe{color:#bbb;font-size:18px;font-weight:100;padding-right:8px}.obrand_loginPageLogo{background-image:url(images/ovirt_logo.png);width:80px;height:30px;border:0px;display:block}.obrand_loginPageLogoLink{display:block;float:right;margin-right:64px;margin-top:75px;margin-bottom:70px;position:relative;text-align:center}@media (max-width: 768px){.obrand_loginPageLogo{display:none}.obrand_loginPageLogoLink{display:none}}.obrand_middleLogoName{background-image:url(images/ovirt_middle_logo.png);width:440px;height:35px;border:0px;display:block;background-size:contain;background-repeat:no-repeat;float:left;clear:left}.obrand_ssoHeader{height:41px;position:relative;top:4px;background-color:#121212}.obrand_main_tab{height:calc(100vh - 66px);min-height:calc(100vh - 66px)}.obrand_detail_tab{min-height:265px}.obrand_landingBgTop{position:absolute;top:0;right:0;background-image:url(images/ovirt_landing_bg_top_right.png);width:561px;height:413px;background-repeat:no-repeat}.obrand_landingBgBottom{background-image:url(images/ovirt_landing_bg_bottom_left.png);position:absolute;bottom:0;left:0;width:100%;height:100%;border-top:4px solid #00659c;background-repeat:no-repeat}.obrand_mastheadLogo{background-image:url(images/ovirt_masthead_logo.png);width:272px;height:16px;position:relative;top:6px;border:0px;display:inline !important;background-size:contain;background-repeat:no-repeat}.obrand_mastheadBackground{background-image:url(images/ovirt_masthead_bg.png);background-size:cover}.obrand_aboutBackground{width:565px;height:355px;background-image:url(images/ovirt_about_bg.png)}.obrand_aboutApplicationLogo{background-image:url(images/ovirt_logo.png);width:80px;height:30px;border:0px;display:block;background-size:contain;background-repeat:no-repeat;float:right;position:relative;top:140px}
```

Comment:
I understand that this is probably output of a less compiler. Could use please an IDE formatting tool to make it look a bit more like source code? I can come handy in case of some web-ui specific changes.

Reply:
copying the scss source now (into ovirt-brand.css) instead of the css output.